cdaringe/css-screen
# Ignore node_modules (😱🚿)

* **Problem:** This repository tracks _node_modules_ directories, which is largely considered bad form in the JavaScript/Node.js community.
* **Solution:** `git rm` _todomvc/node_modules_ and add [GitHub’s Node.js-specific .gitignore file](https://github.com/github/gitignore/blob/master/Node.gitignore)
* **Testing:**
  1. Clone repository and check out branch
  2. Observe no _node_modules_ directory is present
  3. `cd todomvc && npm i`
  4. Ensure git doesn't see new files (`git status` yields “nothing to commit, working tree clean”)
